### PR TITLE
Change to allow multiplex code to be used as a library.

### PR DESCRIPTION
**Pull request by DrMarcII**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -8,10 +8,10 @@ author: Devon Carew <devoncarew@google.com>
 environment:
   sdk: '>=1.10.0 <2.0.0'
 dependencies:
-  args: '^0.13.1'
-  logging: '^0.11.0'
+  args: '^0.13.2'
+  logging: '^0.11.1'
```
Comment (devoncarew) [inline pubspec.yaml]: If it doesn't depend on strictly greater than 0.11.1, let's stick with the 0.11.0 constraint (same with args). That'll reduce the number of downstream version conflicts.

**Comment by devoncarew** [comment]
lgtm w/ a pubspec change

File: pubspec.yaml (hunk 1 above)
Comment (DrMarcII) [inline pubspec.yaml]: Changed, but below is my thinking for how I have been maintaining version constraints:

I always update the version constraints to the exact version of the most recent stable version of whatever package I am using. Any ^... style version constraint that is compatible with the something that leaves .0 at the end will work with the more exact version constraint. However by using the tighter constraint I ensure that users will be using the same or newer version of the package as I tested with. Additionally, a tighter constraint should actually improve the performance of pubs constraint solver (fewer options to consider).

Comment (devoncarew) [inline pubspec.yaml]: Makes sense. Many of the problems I've run into w/ pub have involved incompatible version constraint windows. It's pretty frustrating when you have two packages you depend on that specify separate, incompatible version subranges...

**Pull request merged**